index.html

```diff
@@ -260,6 +260,439 @@ <h1 class="primary-text1">
                 </li>
             </ul>
     </section>
+    <section class="work">
+        <ul class="main-card-container">
+            <li class="card">
+                <img class="snap-img-class" src="resources/Snapshoot Portfolio.svg" alt="resources/Snapshoot Portfolio.svg"></img>
+                <div class="card-block">
+                    <div class="card-header">
+                        <span class="card-header-text">Tonic</span>
```

Comment:
- [ ] This is the project title and a header tag `<hx>` would be better placed here. Please change that. (remember to make the changes in all project cards and substitute the 'x' for the correct number following the headings hierarchy)